GDJS/Runtime/CustomRuntimeObjectInstanceContainer.ts

```diff
@@ -148,8 +148,10 @@ namespace gdjs {
       for (let i = 0, len = allInstancesList.length; i < len; ++i) {
         const object = allInstancesList[i];
         object.onDeletedFromScene(this);
+        // The object can free all its resource directly...
+        object.onDestroyed();
```

Comment:
Should the call to each object.onDestroyed() be done in _destroy?

Reply:
I don't know if it's better or not but `RuntimeScene.unloadScene()` does something similar to this PR changes so I guess it's safer to do the same even if it probably wouldn't change what is executed.
I guess it won't be an issue as long as `RuntimeScene` and `CustomRuntimeObjectInstanceContainer` are not sub-classed.